keywords.md

```diff
@@ -56,6 +56,8 @@ CREATE TABLE test.select (BEGIN int, END int);
 Query OK, 0 rows affected (0.08 sec)
 ```
 
+TiDB provides a list of keywords in the [`INFORMATION_SCHEMA.KEYWORDS`](/information-schema/information-schema-keywords.md) table.
```

Comment:
```suggestion
TiDB v7.6.0 provides a list of keywords in the [`INFORMATION_SCHEMA.KEYWORDS`](/information-schema/information-schema-keywords.md) table.
```

Reply:
Do we need to add a version if this gets merged in `master` and `release-7.6` ?

Reply:
```suggestion
TiDB provides a list of keywords in the [`INFORMATION_SCHEMA.KEYWORDS`](/information-schema/information-schema-keywords.md) table since version 7.6.
```

```suggestion
TiDB provides a list of keywords in the [`INFORMATION_SCHEMA.KEYWORDS`](/information-schema/information-schema-keywords.md) table. (New in v7.6)
```

```suggestion
TiDB v7.6 and newer provides a list of keywords in the [`INFORMATION_SCHEMA.KEYWORDS`](/information-schema/information-schema-keywords.md) table.
```

Reply:
@Oreoxmt Feel free to modify this in any way you think is best.

Reply:
@dveeden I have updated it to "Starting from v7.6.0, TiDB provides..." in https://github.com/pingcap/docs/pull/15795/files#r1432399240